packages/devextreme/testing/testcafe/tests/editors/textArea/index.ts

```diff
@@ -1,101 +1,101 @@
+
+  await testScreenshot(t, takeScreenshot, 'TextArea correct height with autoResizeEnabled, height is 7em & maxHeight is 5em.png', { element: '#container' });
 
-[true, false].forEach((autoResizeEnabled) => {
-  test(`TextArea has correct height with "autoResizeEnabled" is ${autoResizeEnabled} and height is 7em & maxHeight is 5em`, async (t) => {
-    const textArea = new TextArea('#container');
-
-    await t
-      .expect(getPropertyValue(textArea.element, 'maxHeight'))
-      .eql('5em')
-      .expect(getPropertyValue(textArea.element, 'height'))
-      .eql('7em')
-      .expect(textArea.element.getStyleProperty('height'))
-      .eql('70px')
-      .expect(getPropertyValue(textArea.input, 'height'))
-      .eql('');
-  }).before(async () => createWidget('dxTextArea', {
+  await t
+    .expect(compareResults.isValid())
+    .ok(compareResults.errorMessages());
+}).before(async () => {
+  await setAttribute('#container', 'style', 'width: 300px; height: 400px;');
+
+  const config = {
     maxHeight: '5em',
     height: '7em',
-    autoResizeEnabled,
-    width: 200,
-    value: text,
-  }));
-
-  test(`TextArea has correct height with "autoResizeEnabled" is ${autoResizeEnabled} and height is 5em & maxHeight is 7em`, async (t) => {
-    const textArea = new TextArea('#container');
-
-    await t
-      .expect(getPropertyValue(textArea.element, 'maxHeight'))
-      .eql('7em')
-      .expect(getPropertyValue(textArea.element, 'height'))
-      .eql('5em')
-      .expect(textArea.element.getStyleProperty('height'))
-      .eql('70px')
-      .expect(getPropertyValue(textArea.input, 'height'))
-      .eql('');
-  }).before(async () => createWidget('dxTextArea', {
-    maxHeight: '7em',
-    height: '5em',
-    autoResizeEnabled,
-    width: 200,
+    width: '100%',
     value: text,
-  }));
-
-  test(`TextArea has correct height with "autoResizeEnabled" is ${autoResizeEnabled} and maxHeight option is 5em`, async (t) => {
-    const textArea = new TextArea('#container');
-
-    await t
-      .expect(getPropertyValue(textArea.element, 'maxHeight'))
-      .eql('5em')
-      .expect(getPropertyValue(textArea.element, 'height'))
-      .eql(autoResizeEnabled ? 'auto' : '')
-      .expect(textArea.element.getStyleProperty('height'))
-      .eql(autoResizeEnabled ? '70px' : '55px')
-      .expect(getPropertyValue(textArea.input, 'height'))
-      .eql(autoResizeEnabled ? '68px' : '');
-  }).before(async () => createWidget('dxTextArea', {
-    maxHeight: '5em',
-    autoResizeEnabled,
-    width: 200,
+  };
+
+  await appendElementTo('#container', 'div', 'textArea1');
+  await appendElementTo('#container', 'div', 'textArea2');
+
+  await createWidget('dxTextArea', {
+    ...config,
+    autoResizeEnabled: true,
+  }, '#textArea1');
+
+  await createWidget('dxTextArea', {
+    ...config,
+    autoResizeEnabled: false,
+  }, '#textArea2');
+});
+
+test('TextArea has correct height autoResizeEnabled and different value length', async (t) => {
```

Comment:
Since the autoResizeEnabled option cannot be configured outside of the test and we are testing both configurations in a single screenshot, we do not need to specify 'with autoResizeEnabled' in the test name.

```
test('TextArea should have correct height ... ', async (t) => {
```